config/production.json

```diff
@@ -12,6 +12,7 @@
 	"jaeger.sampler.param": 0.001,
 
 	"logger.output": "stdout",
+	"logger.fileName": "",
```

Comment:
hmm, the value being "" seems a bit confusing

Reply:
the default fileName is "" aka no file.

The config library expects keys to exist.